src/drive/mobile/modules/settings/Settings.jsx

```diff
@@ -3,6 +3,8 @@
 import React, { Component } from 'react'
 import { translate } from 'cozy-ui/transpiled/react/I18n'
 import Alerter from 'cozy-ui/transpiled/react/Alerter'
+import { RealTimeQueries } from 'cozy-client'
```

Comment:
nit: est-ce qu'on ne passerait pas maintenant toujourspar des liens profonds pour toutes nos libs ?

Reply:
With Cozy-Client I don't see any args of doing it. The lib is pretty small. Specially when we're already in a react context. 

I don't like to use deep path because to me, internal structure is not an "public api".

Reply:
Pourquoi faire des liens profonds avec `Q` dans ce cas ?